day_7/10_little_monkeys.rb

```diff
@@ -0,0 +1,23 @@
+`10 LITTLE MONKEYS`
+
+for i in (10).downto(1)
```

Comment:
Enumerables in ruby are typically preferred to for loops. How might you change this to use `.each` instead of a for loop?